src/platform/windows/intro.md

```diff
@@ -0,0 +1,13 @@
+# Windows
+
+## Installing & Upgrading Anki on Windows
+
+For instructions on how to install or upgrade Anki on Windows, please see:
+- [Install & Upgrade](installing.md)
+
+## Problems
+
+If you encounter any issues when installing or starting Anki, please see the
```

Comment:
Not all Windows pages are linked to here.

Reply:
@dae suggested to me adding text like "Please see the subsections in the table of contents." in the intro pages instead of listing all subsections.